src/js/background/reopen.js

```diff
@@ -0,0 +1,69 @@
+const reopenIn = {
+  // Map from menuItemId to cookieStoreId.
+  cookieStoreIds: new Map(),
+
+  init() {
+    browser.menus.onShown.addListener(async (info, tab) => {
+      if (info.contexts.length !== 1) {
+        return;
+      }
+      if (info.contexts[0] !== "tab") {
+        return;
+      }
+
+      await this.rebuildMenu(tab);
+    });
+
+    browser.menus.onClicked.addListener((info, tab) => {
+      browser.tabs.create({
+        url: tab.url,
```

Comment:
New tab url is 'about:newtab'. Does not work. I think it needs to be replaced by NULL.
> Error: Illegal URL: about:newtab

Reply:
Indeed, thanks :)
but I think opening other open-able tab is not something we should do, since it contradicts with the menu label.

if we implement this as WebExtension feature, we shouldn't show the tab menu for tabs which WebExtensions cannot reopen.
(about:*, data:*, and maybe chrome:* etc?)

Reply:
> we shouldn't show the tab menu for tabs which WebExtensions cannot reopen.

I agree.